packages/manager/apps/dbaas-logs/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@ovh-ux/manager-dbaas-logs-app",
+  "version": "0.0.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/dbaas-logs"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "yarn build:eu && yarn build:ca && yarn build:us",
+    "build:ca": "webpack --env.production --env.region='CA' && mv dist dist-CA",
+    "build:eu": "webpack --env.production && mv dist dist-EU",
+    "build:us": "webpack --env.production --env.region='US' && mv dist dist-US",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-dbaas-logs-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-dbaas-logs-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-dbaas-logs-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^0.4.0",
+    "@ovh-ux/manager-core": "^8.0.0 || ^9.0.0",
```

```suggestion
    "@ovh-ux/manager-core": " ^9.0.0",
```
now it has been released

---

This has been updated